### ISSUE TITLE
no etherum https://blockchain.info/address/1CRNSDJTJyHBwWVAynkHufmYsteFsmnNLh

### ISSUE THREAD

**Issue by grtt** [ethereum/mist#2064]
#### System information
``````
Version: `0.0.0`
OS & Version: windows/linux/osx
Node type: `eth/geth(default)`
``````

#### Please check the **already existing** issues to keep duplicates at a minimum.
Furthermore several **work-arounds** have been collected in the [Mist-Troubleshooting-Guide](https://github.com/ethereum/wiki/wiki/Mist-Troubleshooting-Guide).

If possible add the following to your report:

- Screenshots

- Check the console, of Mist (`CTRL/CMD + ALT + i`) and take a screenshot

- Log files 
  - go to `menu -> accounts -> backup -> application data`
  - zip and upload `node.log` and all other `node.log.X` files

**Comment by evertonfraga** [comment]
Hello. First of all, is your Mist fully synced? 
In order to confirm your balance, before it has finished to sync, you can hop into a blockchain explorer like etherscan, and search for your ethereum address. 

Example: https://etherscan.io/address/0x33be28a46b6dc3211350a747d821eb6a2ba089de

![image](https://cloud.githubusercontent.com/assets/47108/24358212/cb7d5e04-12d6-11e7-86f3-822f1bcb2d8f.png)

**Comment by luclu** [comment]
Closed as duplicate of #2070.
